datarequest/schemas/youth-0/datarequest/schema.json

```diff
@@ -29,9 +29,13 @@
           "items": {
             "type": "object",
             "properties": {
-              "name": {
+              "first_name": {
```

Comment:
In default-1 we use first name as label, but the field is named given_name. To be consistent with DataCite. And Given name and Family Name may also be a nicer combination.

Reply:
Agreed. Fixed.